openfe/utils/visualization_3D.py

```diff
@@ -106,6 +106,39 @@ def _add_spheres(view:py3Dmol.view, mol1:Chem.Mol, mol2:Chem.Mol, mapping:Dict[i
             }
         )
 
+@requires_package("py3Dmol")
+def show_ComponentCoords(mols: Union[Component, Iterable[Component]],
+                     style: Optional[str] ="stick",
+                     ) -> py3Dmol.view:
+    """this function can be used to visualize multiple component coordinates in one view.
+    It helps to understand how the components are aligned in the system to each other.
+
+    Parameters
+    ----------
+    mols : Union[Component, Iterable[Component]]
+        collection of components
+    style : Optional[str], optional
+        py3Dmol style, by default "stick"
+
+    Returns
+    -------
+    py3Dmol.view
+        view containing all component coordinates
+    """
+
+    if(isinstance(mols, Component)):
+        mols = [mols]
+        
+    view = py3Dmol.view(width=600, height=600)
+    
+    for component in mols:
+        mol = component.to_rdkit()
```

Comment:
isn't this sufficient ducktyping that you don't need the isinstance check?

Reply:
hm... thought it might be convenience, but we can make it leaner.